main/ScaledVersion/Corona.md

```diff
@@ -1,29 +1,53 @@
 ---
 layout: default
 title: Coronas
-subtitle: What's that bright light around that star?
+sidenote: what the hell squad
 ---
 
-The `Coronas { }` node is a wrapper node for separate `Value { }` subnodes that contain information for a corona.
+The `Coronas { }` node is a wrapper node for separate `Corona { }` subnodes that contain information for a corona.
 
 **Example**
 ```
-TODO
+ScaledVersion
+{
+    Coronas
+    {
+        Corona
+        {
+            rotation = 0
+            speed = -1
+            updateInterval = 5
+            scaleLimitX = 0
+            scaleLimitY = 0
+            scaleSpeed = 0
+            
+            Material
+            {
+                texture = TheStarclimberProgram/ConfigFiles/KepsolAndKilter/PluginData/KepsolCorona.png
+                inverseFade = 1
+            }
+        }
+    }
+}
 ```
 
+Through experimentation, it was realized that the only setting really worth changing is Material/texture, as not much improvement can be gotten out of wrangling with the other settings. Because of this, it was decided to omit the description tables. With that in mind, feel free to use the above values apart from the texture path.  
+
+{% comment %}
```

Comment:
Wait what's this?

Reply:
I asked Snails what they did and he said to not bother because you couldn't get any improvement out of them. If i'm telling people not to bother with them, why add the tables?

Reply:
Oh I was wondering how the `comment` block worked lol. Does it just omit the information from the page?

Reply:
It works like `/* */` yeah.